.travis.yml

```diff
@@ -13,4 +18,6 @@ before_script:
         else
             PYTEST_ADDOPTS="-m 'not optional'"
         fi
-script: make PYTEST_ADDOPTS="${PYTEST_ADDOPTS}" test-docker
+script:
+    - docker version
```

Comment:
The reason for `docker version` is to show the docker version that we updated manually.

In my memory, if the script in `install` section does show the stdout in the travis log file.
So, if you want to remove the line, I would just remove the line.

How do you think?